### PR TITLE
Fix worldfactory.cpp i18n

### PR DESCRIPTION
**Pull request by EhNuhc**

### CODE REVIEW

File: src/worldfactory.cpp [hunk 1]
```diff
@@ -1290,9 +1290,9 @@ void worldfactory::draw_worldgen_tabs( const catacurses::window &w, size_t curre
     werase( w );
 
     static const std::vector<std::string> tab_strings = { {
```
Comment (Qrox) [inline src/worldfactory.cpp]:
```suggestion
    const std::vector<std::string> tab_strings = { {
```

File: android/app/build.gradle [hunk 2]
```diff
@@ -94,7 +94,7 @@ unzipDeps.dependsOn makeLocalization
 preBuild.dependsOn unzipDeps
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 26
```
Comment (ZhilkinSerg) [inline android/app/build.gradle]: These changes are unrelated.

Comment (ZhilkinSerg) [reply android/app/build.gradle]: You need to revert changes made in gradle file. If you want them merged - they should go to separate PR.

**Comment by BrettDong** [comment]
The issue fixed here has been addressed in my #35420 .

`translate_marker` is intended here to signal `msgfmt` to recognize the string as to be translated while preserving the original string in the `static const` array and call `gettext` function in run-time when drawing UI. If you put `gettext` function straight in initialization of the `static const` array, the content inside will not be able to follow a change of game language.

**Comment by ZhilkinSerg** [comment]
Obsoleted by #35420. Android related changes should go in a separate PR.
